src/qt/pivx/coldstakingwidget.cpp

```diff
@@ -382,16 +382,25 @@ void ColdStakingWidget::updateDisplayUnit() {
     }
 }
 
-void ColdStakingWidget::showList(bool show){
+void ColdStakingWidget::showList(bool show)
+{
```

Comment:
Again, would like to see arbitrary style changes in their own commit, same as my comment in #1389.

Reply:
yup was already going to factor them out in their own commit here too, after reading your comment in the other PR :+1: